examples/rpi/pico2/README.md

```diff
@@ -0,0 +1,45 @@
+# Overview
```

Comment:
let's make a web doc as well? it can be same content. Add a picture or gif too :)